extensions/ql-vscode/src/pure/bqrs-cli-types.ts

```diff
@@ -115,7 +115,7 @@ export type BqrsKind =
   | "Entity";
 
 interface BqrsColumn {
-  name: string;
+  name?: string;
```

Comment:
This is dependent on the query. In the query we're now using, the BQRS column definitions look like this:

```js
[
  { name: "usage", kind: "Entity" },
  { name: "apiName", kind: "String" },
  { kind: "String" },
  { kind: "String" },
]
```

CodeQL is probably not naming the last two columns because it can't automatically determine a name based on the name of a variable:

```ql
usage, apiName, supported.toString(), "supported"
```